lib/pending_assigns_reminder.rb

```diff
@@ -4,7 +4,7 @@ class PendingAssignsReminder
   REMINDED_AT = "last_reminded_at"
   REMINDERS_FREQUENCY = "remind_assigns_frequency"
   CUSTOM_FIELD_NAME = "assigns_reminder"
-  REMINDER_THRESHOLD = 2
+  REMINDER_THRESHOLD = SiteSetting.pending_assign_reminder_threshold
```

Comment:
I don't think you can set the const here, as you cannot change the site setting after the server spin up. I think removing the const, and calling `SiteSetting` directly where the const is  currently used should work.